src/core/bpf-foreign.h

```diff
@@ -5,7 +5,7 @@
 #include "unit.h"
 
 static inline int bpf_foreign_supported(void) {
```

Comment:
Just drop this one?

Reply:
I thought so, but for compatibility with other modules.